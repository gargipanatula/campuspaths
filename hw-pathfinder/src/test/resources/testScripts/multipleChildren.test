CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 1.0
AddEdge simpleGraph A C 1.0
AddEdge simpleGraph A D 1.0
AddEdge simpleGraph D B 1.0