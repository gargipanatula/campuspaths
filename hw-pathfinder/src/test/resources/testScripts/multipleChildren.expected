created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 1.000 from A to B in simpleGraph
added edge 1.000 from A to C in simpleGraph
added edge 1.000 from A to D in simpleGraph
added edge 1.000 from D to B in simpleGraph